vm/src/vm/runners/cairo_runner.rs

```diff
@@ -756,12 +761,39 @@ impl CairoRunner {
         Ok(())
     }
 
+    ///Relocates the VM's trace, turning relocatable registers to numbered ones
+    fn relocate_trace(
```

Comment:
Should we make this functions public?
Since the deleted method `pub fn relocate_trace(&mut self, relocation_table: &[usize]) -> Result<(), TraceError>` was public?

Reply:
we can